src/lib/analytics/churnPrediction.ts

```
@@ -0,0 +1,55 @@
+    }
+
+    return data as ChurnRiskAssessment;
+}
+
+/**
+ * Fetches all attendees for a series along with their current churn risk assessments.
+ */
+export async function getSeriesAttendeesWithChurnRisk(eventSeriesId: string): Promise<SeriesAttendeeRisk[]> {
+    const { data, error } = await supabase
+        .from('user_series_churn_risk')
+        .select(`
```

ESLint

[error] Replace `········.select(` with `····.select(⏎······`

(prettier/prettier)